docs/antora/modules/ROOT/pages/glossary.adoc

```diff
@@ -17,11 +17,14 @@ a|*Conceptual Mapping* (often abbreviated as CM)
 |*eForms*
 |eForms is the notification standard used for publishing public procurement procedures notices in the European Union, which replaces the Standard Forms (see below).  eForms are used to publish public procurement notices according to Regulation (EU) 2019/1780. For more information on this, see the https://docs.ted.europa.eu/eforms/latest/index.html[eForms SDK documentation].  The eForms conform to the TED SDK.
 
+a|*Cellar*
+a|Cellar is the Semantic Repository of the EU Publications Office, also referred to as the Triple Store, from where data can be reused via Sparql query using the https://publications.europa.eu/webapi/rdf/sparql[Sparql Endpoint].
```

Comment:
Cellar is a document system and the triplestore is only part of it. To avoid confusion, use Cellar triplestore